projects/ghostscript/gstoraster_fuzzer.cc

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <base/gserrors.h>
+#include <psi/iapi.h>
+
+#include <limits.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+
+static const unsigned char *g_data;
+static size_t g_size;
+
+#define min(x, y) ((x) < (y) ? (x) : (y))
+
+static int gs_stdin(void *inst, char *buf, int len)
+{
+	size_t to_copy = min(len, g_size);
+	to_copy = min(INT_MAX, to_copy);
+
+	memcpy(buf, g_data, to_copy);
+
+	g_data += to_copy;
+	g_size -= to_copy;
+
+	return to_copy;
+}
+
+static int gs_stdout(void *inst, const char *buf, int len)
+{
+	/* Just discard everything. */
+	return len;
+}
+
+static int gs_to_raster_fuzz(const unsigned char *buf, size_t size)
+{
+	int ret;
+	void *gs;
+
+	/* Mostly stolen from cups-filters gstoraster. */
+	char *args[] = {
+		"-r200x200",
+		"-dMediaPosition=1",
+		"-dcupsColorSpace=1", /* RGB */
+		"-dQUIET",
+		"-dPARANOIDSAFER",
+		"-dNOPAUSE",
+		"-dBATCH",
+		"-dNOINTERPOLATE",
+		"-dNOMEDIAATTRS",
+		"-sstdout=%stderr",
+		"-sOutputFile=%stdout",
+		"-sDEVICE=cups",
+		"-_",
```

Comment:
Should this include `-dSAFER`?

Reply:
@alex this has -dPARANOIDSAFER, which I think should cover the same behavior.